### PR TITLE
Issue 12 gif generator

### PR DESCRIPTION
**Pull request by IxLahiji**
Implements issue #12.

### CODE REVIEW

**Comment by IxLahiji** [comment]
@KeithPetro @AtriLahiji If you could review the pull request, that would be great.

File: GrepAdminBot/GrepAdminBot/Modules/Public.cs [hunk 1]
```diff
@@ -60,5 +66,58 @@ private string Coin()
...
         }
+
+        [Command("gif"), Priority(0)]
+        [Summary("Posts a random gif.")]
+        public async Task Gif()
+        {
+            string giphyToken = this.config["tokens:giphy"];     // Get the discord token from the config file
+            if (string.IsNullOrWhiteSpace(giphyToken))
+            {
+                await ReplyAsync("No Giphy app token provided. Please enter token into the `_config.json` file found in the applications root directory.");
```
Comment (KeithPetro) [inline GrepAdminBot/GrepAdminBot/Modules/Public.cs]: Config file is "config.json", not "_config.json"

File: GrepAdminBot/GrepAdminBot/Modules/Public.cs [hunk 2]
```diff
@@ -60,5 +66,58 @@ private string Coin()
...
+        }
+
+        [Command("gif"), Priority(1)]
+        [Summary("Posts a gif realted to the description provided.")]
+        public async Task Gif([Remainder]string query)
+        {
+            string giphyToken = this.config["tokens:giphy"];     // Get the discord token from the config file
+            if (string.IsNullOrWhiteSpace(giphyToken))
+            {
+                await ReplyAsync("No Giphy app token provided. Please enter token into the `_config.json` file found in the applications root directory.");
```
Comment (KeithPetro) [inline GrepAdminBot/GrepAdminBot/Modules/Public.cs]: Config file is "config.json", not "_config.json"

**Review by KeithPetro — COMMENTED** [review]
Search query which results in 0 gifs being found will throw a DivideByZeroException. This can seemingly be fixed by handling the case where there are zero search results within the `gif` command of `priority 1`.

**Comment by IxLahiji** [comment]
Fixed the issues, could I get another review?

**Review by KeithPetro — APPROVED** [review]
Everything looks to be good now.

**Pull request merged**